esphome/ds18b20_beta.yaml

```
@@ -0,0 +1,36 @@
+---
+# vindriktning 
+#
+# a simple ESPHome integration to make an vindriktning 
+# © Maarten Schmeitz
+#
+substitutions:
+  device_name: "ds18b20"
+  
```

yamllint

[error] trailing spaces

(trailing-spaces)